src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -65,6 +65,12 @@
     public static final String GENERAL_LEDGER_PENDING_ENTRIES_TITLE = "General Ledger Pending Entries";
     public static final String LEFT_PARENTHESIS = "(";
     public static final String RIGHT_PARENTHESIS = ")";
+    
+    public static final String CORPORATE_BILLED_CORPORATE_PAID_DOCUMENT_TYPE_CODE = "CBCP";
+    public static final String CORPORATE_BILLED_CORPORATE_PAID_ERROR_EMAIL_SUBJECT = "Error occurred during Corporate Billed Corporate Paid batch upload process";
```

Comment:
Should these two errors be moved to CU-ApplicationResources.properties?

Reply:
sure